src/main/webapp/script.js

```diff
@@ -70,13 +142,20 @@ class Measure {
         parentDiv.appendChild(dropDownDiv);
         dropDownDiv.appendChild(buttonDrop);
         dropDownDiv.appendChild(innerDrop);
-        innerDrop.appendChild(Piano_A);
-        innerDrop.appendChild(Piano_B);
+
+        // adding the sound options we created to the drop down
+        innerDrop.appendChild(sound_option_1);
+        innerDrop.appendChild(sound_option_2);
+        innerDrop.appendChild(sound_option_3);
+        innerDrop.appendChild(sound_option_4);
+        innerDrop.appendChild(sound_option_5);
         dropDownDiv.appendChild(buttonMusic);
 
     }
 
-    pianoA() {
+    // TODO: edit the pianoA and pianoB method to set the sound chosen as the sound for current note 
+    pianoA(file) {
+        console.log(file);
```

Comment:
Do you need to pass "file" as a parameter? I believe you want to pass "file" as an argument to the Note class constructor.